tests/qos/test_qos_sai.py

```diff
@@ -1842,7 +1883,9 @@ def testQosSaiDwrrWeightChange(
     @pytest.mark.parametrize("queueProfile", ["wm_q_wm_all_ports"])
     def testQosSaiQWatermarkAllPorts(
         self, queueProfile, ptfhost, dutTestParams, dutConfig, dutQosConfig,
-        resetWatermark, _skip_watermark_multi_DUT
+        get_src_dst_asic_and_duts, _skip_watermark_multi_DUT,
+        skip_pacific_dst_asic, dut_qos_maps,  # noqa F811
```

Comment:
ERROR qos/test_qos_sai.py::TestQosSai::testQosSaiQWatermarkAllPorts[single_asic-wm_q_wm_all_ports]
file /azp/_work/22/s/tests/qos/test_qos_sai.py, line 1883
      @pytest.mark.parametrize("queueProfile", ["wm_q_wm_all_ports"])
      def testQosSaiQWatermarkAllPorts(
E       fixture 'skip_pacific_dst_asic' not found

Reply:
Thanks for catching it. I will remove this for now, since it is to be coming from the other PR.